packages/mobile/src/transactions/saga.ts

```diff
@@ -16,19 +19,30 @@ import {
   removeStandbyTransaction,
   transactionConfirmed,
   transactionFailed,
+  updateRecentTxRecipientsCache,
 } from 'src/transactions/actions'
 import { TxPromises } from 'src/transactions/contract-utils'
-import { standbyTransactionsSelector } from 'src/transactions/reducer'
+import {
+  knownFeedTransactionsSelector,
+  KnownFeedTransactionsType,
+  standbyTransactionsSelector,
+} from 'src/transactions/reducer'
 import { sendTransactionPromises, wrapSendTransactionWithRetry } from 'src/transactions/send'
 import { StandbyTransaction, TransactionStatus } from 'src/transactions/types'
 import Logger from 'src/utils/Logger'
 
 const TAG = 'transactions/saga'
 
+const RECENT_TX_RECIPIENT_CACHE_LIMIT = 10
```

Comment:
what led you to this number? just based on screen size, storage, the work being expensive?

Reply:
It was a combination of screen size and storage considerations. 10 tx feed items is what's viewable without scrolling on a pretty large phone. We could always store more but wanted to be mindful of storage.

Do you think the number should be different?

Reply:
Sounds reasonable - was just curious